backend/curriculum_tracking/admin.py

```diff
@@ -176,6 +180,49 @@ def bulk_deactivate_users(self, request, users):
 
     bulk_deactivate_users.short_description = "Deactivate selected users"
 
+    def invite_to_github_repos(self, request, users):
+        permitted_teams_for_users = self._get_permitted_teams_for_users(users)
+        if request.POST.get("post", None):
+            from long_running_request_actors import (
+                invite_collaborators_for_team_projects as actor,
+            )
+
+            for team_name in permitted_teams_for_users:
+                actor.send_with_options(kwargs={"team_name": team_name})
+            messages.add_message(
+                request,
+                messages.INFO,
+                f"Adding user(s) as collaborators in the background",
+            )
+        else:
+            opts = self.model._meta
+            request.current_app = self.admin_site.name
+
+            return TemplateResponse(
+                request,
+                "admin/invite_to_github_repos_confirm.html",
+                {
+                    **self.admin_site.each_context(request),
+                    "title": _("Are you sure?"),
+                    "opts": opts,
+                    "app_label": opts.app_label,
+                    "permitted_teams_for_users": permitted_teams_for_users,
+                    "queryset": users,
+                    "action_checkbox_name": ACTION_CHECKBOX_NAME,
+                },
+            )
+
+    def _get_permitted_teams_for_users(self, users) -> list:
+        teams = []
+        for user in users:
+            for team in get_objects_for_user(
+                user,
+                core_models.Team.PERMISSION_REPO_COLLABORATER_AUTO_ADD,
```

Comment:
This is strange. Why is there a whole new permission here?  It really does not seem necessary

Reply:
Not a new perm, [ref](https://github.com/Umuzi-org/Tilde/blob/10e376e5828766b19387270b2f86f8029ddbfa9c/backend/core/models.py#L253).

snippet:
```python
class Team(AuthGroup, Mixins):

    PERMISSION_MANAGE_CARDS = PERMISSION_MANAGE_CARDS  # start cards, set due date,
    PERMISSION_VIEW_ALL = PERMISSION_VIEW_ALL  # look at anything
    PERMISSION_ASSIGN_REVIEWERS = PERMISSION_ASSIGN_REVIEWERS
    PERMISSION_REVIEW_CARDS = PERMISSION_REVIEW_CARDS
    PERMISSION_TRUSTED_REVIEWER = PERMISSION_TRUSTED_REVIEWER

    PERMISSION_VIEW = [
        # anyone with these permissions get to navigate to a specific team on the frontend and see the status of all the cards.
        PERMISSION_MANAGE_CARDS,
        PERMISSION_VIEW_ALL,
        PERMISSION_ASSIGN_REVIEWERS,
        PERMISSION_REVIEW_CARDS,
        PERMISSION_TRUSTED_REVIEWER,
    ]

    PERMISSION_REPO_COLLABORATER_AUTO_ADD = [
        # when a repo is created then folks with these permissions get added to the repo as collaborators
        PERMISSION_MANAGE_CARDS,
        PERMISSION_VIEW_ALL,
        PERMISSION_ASSIGN_REVIEWERS,
        PERMISSION_REVIEW_CARDS,
    ]
```